content/id/docs/concepts/policy/pod-security-policy.md

```diff
@@ -0,0 +1,469 @@
+---
+title: Pod Security Policy
+content_template: templates/concept
+weight: 20
+---
+
+{{% capture overview %}}
+
+{{< feature-state state="beta" >}}
+
+Pod Security Policies (kebijakan keamanan Pod) memungkinkan otorisasi secara detil dari pembuatan dan pembaruan Pod.
+
+{{% /capture %}}
+
+{{% capture body %}}
+
+## Apa itu Pod Security Policy?
+
+_Pod Security Policy_ adalah sebuah sumber daya pada tingkat kluster yang mengatur aspek-aspek spesifikasi Pod yang sensitif terhadap keamanan. Objek-objek `PodSecurityPolicy` mendefinisikan sebuah kumpulan kondisi yang harus dijalankan oleh Pod untuk dapat diterima oleh sistem, dan juga sebagai nilai-nilai bawaan untuk kolom-kolom yang bersangkutan. Mereka memungkinkan administrator untuk mengatur hal-hal berikut:
+
+| Aspek yang diatur                                      | Nama Kolom                                 |
+| ----------------------------------------------------| ------------------------------------------- |
+| Menjalankan Container-container yang _privileged_   | [`privileged`](#privileged)                                |
+| Penggunaan _namespace-namespace_ milik _host_       | [`hostPID`, `hostIPC`](#host-namespaces)    |
+| Penggunaan jaringan dan _port_ milik _host_         | [`hostNetwork`, `hostPorts`](#host-namespaces) |
+| Penggunaan jenis-jenis Volume                       | [`volumes`](#volumes-and-file-systems)      |
+| Penggunaan _filesystem_ milik _host_                | [`allowedHostPaths`](#volumes-and-file-systems) |
+| _White list_ untuk _driver-driver_ Flexvolume       | [`allowedFlexVolumes`](#flexvolume-drivers) |
+| Mengalokasi FSGroup yang memiliki Volume milik Pod  | [`fsGroup`](#volumes-and-file-systems)      |
+| Mengharuskan penggunaan _read-only root filesystem_ | [`readOnlyRootFilesystem`](#volumes-and-file-systems) |
+| User dan Grop ID dari Container                    | [`runAsUser`, `runAsGroup`, `supplementalGroups`](#users-and-groups) |
+| Membatasi eskalasi ke kemampuan _root_              | [`allowPrivilegeEscalation`, `defaultAllowPrivilegeEscalation`](#privilege-escalation) |
+| Kemampuan-kemampuan Linux                           | [`defaultAddCapabilities`, `requiredDropCapabilities`, `allowedCapabilities`](#capabilities) |
+| Konteks SELinux dari Container ainer                | [`seLinux`](#selinux)                       |
+| Jenis tambatan Proc yang diizinkan untuk Container  | [`allowedProcMountTypes`](#allowedprocmounttypes) |
+| Profil AppArmor yang digunakan oleh Container       | [annotations](#apparmor)                    |
+| Profil seccomp yang digubakan oleh Container        | [annotations](#seccomp)                     |
+| Profil sysctl yang digunakan oleh Container         | [`forbiddenSysctls`,`allowedUnsafeSysctls`](#sysctl)                      |
+
+## Mengaktifkan Pod Security Policy
```

Comment:
Should we use a Kebijakan _Security_ or retaining Security Policy?

Reply:
I think we should retain it, as it refers to the API Object, to prevent confusion.

https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.16/#podsecuritypolicy-v1beta1-policy